composer.json

```diff
@@ -98,10 +98,6 @@
                 "Ensure field definition allowed values callbacks are used for field filter callbacks": "https://www.drupal.org/files/issues/2020-06-03/2949022-12--views_filter_options_callback.patch",
                 "Selecting the same day in a date between filter returns no results": "https://www.drupal.org/files/issues/2021-05-05/date_between_2842409_2_d8.patch"
             },
-            "drupal/socialbase": {
```

Comment:
I'm not sure why this is part of your branch, shouldn't be related to this work right? 🤔